rasterio/mask.py

```diff
@@ -5,10 +5,8 @@
 
 import numpy
 
-import rasterio._loading
-with rasterio._loading.add_gdal_dll_directories():
-    from rasterio.errors import WindowError
-    from rasterio.features import geometry_mask, geometry_window
```

Comment:
The main branch has a change that conflicts with the above. Rebasing on main would be a good idea.